python/sqlalchemy/src/rest-service.py

```diff
@@ -0,0 +1,276 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import logging
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker, relationship
+from http.server import HTTPServer, BaseHTTPRequestHandler
+import json
+from io import BytesIO
+import jsonpickle
+from yugabyte_alchemy_model import User, Order, OrderLine, Product
+
+
+logging.basicConfig(
+    level=logging.DEBUG,
+    format="%(asctime)s:%(levelname)s:%(message)s"
+    )
+
+logging.getLogger('sqlalchemy.engine.base.Engine').setLevel(logging.WARNING)
+
+listen_port = 8000
```

Comment:
Is  it possible to move these into a config file? Instructions to customize become a lot easier. Ok with this being a python file, and we import that class here.

Reply:
Yes, it makes it clearer.  Done.